roles/gitops/vault-secrets/vars/post-install.yaml

```
@@ -39,7 +39,14 @@ envs:
               secret: "{{ vault_client_secret | default('') }}"
       - argocd_app: "vault"
         vault_values:
-          vaultToken: "{{ vault_token | default('') }}"
+          backupSecrets:
+            vaultToken: >-
+              {{
+                (vault_keys.resources[0].data.root_token)
+                if vault_keys.resources[0].data.root_token is defined and
+                vault_keys.resources[0].data.root_token | length > 0
+                else ''
+              }}
```

Déjà présent dans le defaults/main.yaml il y a un interet à les redéclarer ici ?

---

Bonne remarque, lui à la base n'était présent qu'en post-install parce qu'on ne pouvait pas le récupérer avant que le vault ne soit déployé. Mais on aura besoin de le mettre dans `defaults/main.yaml` dans le futur (initialisé avec une valeur vide si besoin) pour que l'application soit healthy afinq que le job de post-configuration puisse se déclencher (PostSync hook).
Par contre, il faut respecter l'ancien chemin.
```suggestion
          vaultToken: >-
            {{
              (vault_keys.resources[0].data.root_token)
              if vault_keys.resources[0].data.root_token is defined and
              vault_keys.resources[0].data.root_token | length > 0
              else ''
            }}
```